src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -831,9 +831,9 @@ export default class AircraftModel {
         const waypointSpeed = waypointModel.speedMaximum;
         const waypointDistance = this.positionModel.distanceToPosition(waypointModel.positionModel);
         const speedChange = waypointSpeed - this.speed;
-        const decelerationRate = -this.model.rate.decelerate / 2;   // units of rate.decel are 'knots per 2 seconds'
+        const decelerationRate = -this.model.rate.decelerate / 2000;   // units of rate.decel are 'knots per 2 seconds, in milliseconds'
```

Comment:
Has rate.decel changed units?  Looking in the code base I think decelrate is always given in knots per second, so we should keep the old comment